### DISCUSSION TITLE
Read More Option in Testimonial Carousel

### DISCUSSION THREAD

**Discussion by naikoosuhaib** [elementor/elementor#24331] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Many individuals commonly utilize the Google Widget to automatically retrieve reviews from their Google Page. However, challenges arise when it comes to manually inputting testimonials, especially if the length of one testimonial differs significantly from another.

So, I actually want that a "Read More" feature is added so that we can limit either words of each testimonial so that they look good. 

### Describe the solution you'd like

I'd like to propose the inclusion of a "Read More" feature to limit the number of words in each testimonial. This will contribute to a more polished and visually pleasing website layout, especially on mobile devices where disparate testimonial lengths can otherwise lead to awkward empty spaces, detracting from the overall aesthetic appeal.

**Comment by ravenstudios-bfoster** [comment]
I would love this feature as well. This would be a great addition!

**Comment by nicholaszein** [comment]
As a workaround, you could use the new nested Carousel widget (Pro), build your own layout, and include a button widget below each testimonial. This way you have all the flexibility and liberty of building your own layouts. 😉
